spec/index.bs

```diff
@@ -4238,20 +4238,25 @@ dictionary GPUBlendComponent {
 enum GPUBlendFactor {
     "zero",
     "one",
-    "src-color",
-    "one-minus-src-color",
-    "src-alpha",
-    "one-minus-src-alpha",
-    "dst-color",
-    "one-minus-dst-color",
+    "src0-componentwise",
+    "one-minus-src0-componentwise",
+    "src0-alpha",
+    "one-minus-src0-alpha",
+    "dst-componentwise",
+    "one-minus-dst-componentwise",
     "dst-alpha",
     "one-minus-dst-alpha",
-    "src-alpha-saturated",
-    "blend-color",
-    "one-minus-blend-color"
```

Comment:
Reviewers: please make sure that the naming "componentwise" actually makes sense. It matches my understanding but I'm not 100% sure.

Reply:
It does match, but we can't we just say "source component" and that would be clear enough? I'm not sure what `wise` helps with: it seems more confusing than anything else.

Reply:
Just `component` would be fine with me

Reply:
spitballing: could also be more explicit and say, like, `-rgba` and `-aaaa`/`-a` or something.

Reply:
also here to say that "component" would be nicer to have than "componentwise"